doc/uml/draw-io-classes-doc.xml

```
@@ -0,0 +1 @@
UxPZv5vttXkWeeija0mCTDVF4fgAE0eyzZBEHW2CfyfU7wjF6dgvasqU7DKDeUGoJQNc5ZnIAL8s04BR2Mo3ptBoibaEBej4+jH7BasEcmZJV/cWJGLqJQU98uUXwpoZdimJBO0wMk/9xUWIQPokuYtDgoRj9os0VYtg7KxMxdELD6I+NE4lBh1diqBaDIs8zIkYwC/HHf0T73HTtouF3cXSOwPCVDT9ZDIapMEzyEQ4qWB7TkU0y9pTHZ+TjzK4XjnKLJaDQYYM2BukbkQJjkNSGckjDz9AXkzIvnk+oSs25ntSIZdaSThqpIggNlKDVGMnBo6UVmurafnEJFgUTVxFhWQFdJoA4A6WGkNOQGhlNapcO1bdIrdgzupLiwTp7SPGq9s8A3QPTyqbgelmOQwyL1ofGcBiJK8o+7mgiViN+FxHxVcRJwNE9yKMYqHSwOElzXxVbzmgiU2NxOkXG3oM0t4Ys+e33UyyeIsBd02F0VeZtZGzWBYUce6DMYeDGUGg8mf9/mcDXWmKrZN0UdDmHAwR9lKutS3C19QmMqCs6LwTZHWawTY2h4oVVgR4UL7XMC0lXDW983ZUYda3ErkUHL8TYdbvqJ9WRNQpQzWn7ygLs0EUM5s9A+R3Z9VQG1ig+NTntK4uv0xJ2x1LAtqE1inBcf7W0jKt2cW4lF/YTrtpO+bGDgq9azc7nWmlBzw7Dw0FDV1pidim3Mq5GYIxboaYZGFcRudlpBYmFs2IEDQ5eGSaek/1FjFMuohEO4wY9vj4cMPi6c3tsPBy7ikZAXCik8nUBQRxzfII4dhWNgFDjLf983WxPTE1Cf5kMhINYPSENidr1XHmnCHReIv2ZqluMhayR2Dbf7tMZIwGEmXxzgQKFKbGgioJSjz5lujNtqUITdYZTAuJ4Wm7jcpelfJkN7V3ooKFolap+flsakaw3LXMLJzXvUHM6WsWZoaHskKphASHdJ7aZlvMRl3PLmSvF6T3feqHjJWRU9flrOj3bFynk76+4efr2dAM/H6Y6ph4aP1odk9nhm/SWvergC9kD3Nmk8iOu7VRReWK15wiPtbirjvwhpRemfFFaNGl8D40f7R1kWtZ41WEvNP60qoFJ40+pELBsMlzfNYjdVt12qe9vUuqTvg/V99Fcdx1vkSjel6fa9PzUOqFJ40/ReJsQxawvKgEuEagrCKeHxsO9CKs+g8TNpPY9f82FnlKgkSJ29oGI3VQZOOIa7zQ13sV9I/Ma6oHtk+J1ZcnfpORdSj7eG6I0FALbI6fxlMnOs/57rp8lnV2sv3VdtAbmr2PHxF2pkuaIiVo2VQUKk1nwM+B7zVc88sJv+9a77DdDMr9FoGXFgt0sWagktttWTdzw5b8sTT8kZLzXlAttLa/8yIWOd6fuVTO47JZKT7tvDldvUPRWoQvVC/wdzRGy63pEnLKhd8FATb3oFdXLoc0cFDww2w27htkYarz6EEd1h7TKu6KcLHzhu6peZg1woFgku+R8Hcp0je4KIDJQwrQBFtdxewn4Ns8cKLvJ9f/wE1C3WWokM0YP35rTdNIQOnKGS5vXsI9do5G377oNGOcPOhjUqqunFdQHC90qqwZqlkNhZJ7FboU5exVNsET2dZZYJbOo6hLPgnfV4wP1QlzGsQwLExtZloGayYPCoUD7T0P9NLLiuBUZVZg010B8+hp4HW7EzQ2b5qviT1jvcCe4HHNe5vXtk8pHkPpfqZDDLpyRgoPtoYUbFOPukTRKaOR0/KOuxCkmS+0VEtd8C4hdEGUZhRlauUwbbxN2i/nXbLeot2zWjJDzLYSrJ/KjuevUHcu5YZCzLATjiuEFYtudtc7EcWsZ+QNfZ02oCQ55R5p351U0EtHIef8VlJhGw440rLNQQq9oRxLH6nwvCkUDYeFYuLm5hJu3p1H64wZJPpmh0CjYbagcsvFAmdl20zdt/sqqwRKDrzHnabV77G3XT9xnosf/AA==</diagram></mxfile>
```

How do we see that?

---

Using draw.io (see file name and `editor="www.draw.io"` inside)

---

Please add a white background to the diagram; Chrome default background for PNG is black thus masking the inheritance relationships

---

Use Firefox.

---

Nianiania

---

Can this be written in a place that will remain easily visible when the PR will be closed?

---

We will create some documentation about this